eng/native/configurecompiler.cmake

```diff
@@ -700,6 +700,9 @@ if (CLR_CMAKE_HOST_UNIX)
 endif(CLR_CMAKE_HOST_UNIX)
 
 if(CLR_CMAKE_TARGET_UNIX)
+  # Enable PAC
+  add_compile_options(-mbranch-protection=pac-ret)
```

Comment:
This needs to be added conditionally for Arm64 only (`CLR_CMAKE_HOST_ARCH_ARM64`).

Reply:
Done